npm install fails due to react-helmet-async peer dependency conflict with React 19

### DESCRIPTION
> On a fresh clone of the repository, running `npm install` fails due to a peer dependency conflict.
> The project currently depends on React 19.x. However, `react-helmet-async@2.0.5` specifies peer support only for React versions ^16.6.0, ^17.0.0, and ^18.0.0. As a result, npm throws an ERESOLVE error and halts installation.
> This prevents contributors from setting up the project locally without using `--force` or `--legacy-peer-deps`, which may lead to unstable dependency resolution.

### Steps to reproduce
1. Clone the repository  
2. Navigate to the frontend directory  
3. Run `npm install`

### Actual behavior
npm fails with an ERESOLVE error related to react-helmet-async peer dependency resolution.

### Expected behavior
The project should install successfully without requiring forced dependency resolution flags.
 

<img width="1486" height="643" alt="Image" src="https://github.com/user-attachments/assets/a637eadc-5222-4d97-8653-0060a17b196d" />

I noticed this issue while setting up the project locally.
@santanu-atta03 I’d be happy to work on a fix if this aligns with the project’s expectations.

--- a/README.md
+++ b/README.md
@@ -205,7 +205,21 @@ npm install
 npm run dev  
 
 ---
+### ⚠️ React Version Compatibility Note
 
+This project currently uses **React 19**.
+
+Some dependencies do not yet officially support React 19.  
+As a result, running `npm install` may fail with an `ERESOLVE` peer dependency error.
+
+#### Temporary Workaround
+
+Until full React 19 support is available across dependencies, install frontend packages using:
+
+```bash
+npm install --legacy-peer-deps
+
+---
 ### 💻 Frontend Setup
 cd frontend  
 npm install  

